modules/platform/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/web/common/requestcontroller/service/FilterConfigDescriptor.java

```diff
@@ -34,11 +37,13 @@
  * @author ldoguin
  */
 @XObject(value = "filterConfig")
+@XRegistry(compatWarnOnMerge = true)
 public class FilterConfigDescriptor {
 
     public static final String DEFAULT_CACHE_DURATION = "3599";
 
     @XNode("@name")
+    @XRegistryId
     protected String name;
```

Comment:
`getName()` defaults to the pattern so we have to be careful here maybe (unless it's a feature nobody uses)

Reply:
Not sure it's used but thanks for catching this, i added pattern as a fallback on the annotation directly :)